content/spin/contributing-docs.md

```diff
@@ -405,6 +405,21 @@ $ bart check content/about.md
 ✅ content/about.md
 ```
 
+If you are [using shortcodes](/bartholomew/shortcodes) and need to tell the `bart check` command where to find them (i.e. if you have your reusable snippets, written in `.rhai` files, tucked away in a relative directory called `my-shortcodes`), you can use the `--shortcodes` option. For example:
+
+```bash
+<!-- @selectiveCpy -->
+
+```bash
+$ bart check --shortcodes ./my-shortcodes content/spin/variables.md
```

Comment:
I would show the actual directory for this site here.  Contributors don't need options, they just need to know what you want _them_ to do.  Minimal friction.

```suggestion
$ bart check --shortcodes ./shortcodes content/spin/variables.md
```